.github/workflows/destroy.yml

```diff
@@ -0,0 +1,18 @@
+
+name: destroy fly app
+on:
+  push:
+    branches:
+      - main
```

Comment:
Any reason why this is running on push to main?

Reply:
oh!. that's a mistake. i'll make the change and push it soon.

Reply:
i have made the necessary change